### ISSUE TITLE
Finetuning Language Model

### ISSUE THREAD

**Issue by ayushidalmia** [ThilinaRajapakse/simpletransformers#700]
**Describe the bug**
I am trying to fine tune a language model using data from my domain. The data is written in text file with one example per line. However, train_model creates the following error:

**To Reproduce**
```
model_args = LanguageModelingArgs()
model_args.output_dir = 'mylocaldata'
model_args.best_model_dir = 'mylocaldata/best_model/'
model_args.reprocess_input_data = True
model_args.overwrite_output_dir = True
model_args.num_train_epochs = 1
model_args.dataset_type = "simple"
model_args.sliding_window = True

model = LanguageModelingModel(
    "bert", "bert-base-uncased", args=model_args
)
# Train the model
model.train_model(train_file, eval_file=test_file)
```

**Screenshots**
This is the error stack:

```
Traceback (most recent call last):
  File "main_simpletransformer.py", line 142, in <module>
    model.train_model(train_file, eval_file=test_file)
  File "/opt/textiq/privacy/miniconda3/lib/python3.8/site-packages/simpletransformers/language_modeling/language_modeling_model.py", line 364, in train_model
    global_step, tr_loss = self.train(
  File "/opt/textiq/privacy/miniconda3/lib/python3.8/site-packages/simpletransformers/language_modeling/language_modeling_model.py", line 566, in train
    outputs = model(**inputs)
TypeError: BertForMaskedLM object argument after ** must be a mapping, not Tensor`
```

**Comment by MirjamOdile** [comment]
This seems to be the same issue as https://github.com/ThilinaRajapakse/simpletransformers/issues/685

**Comment by MirjamOdile** [comment]
Hi @ayushidalmia 

There seems to be a workaround for this issue by turning off fp16 (see issue https://github.com/ThilinaRajapakse/simpletransformers/issues/646) in the model_args. Maybe this helps as a temporary solution.

Cheers
Mirjam

**Comment by ThilinaRajapakse** [comment]
Should be resolved in `v0.48.13`
